tests.sh

```diff
@@ -391,7 +391,7 @@ build_php5.6_c() {
 build_php5.6_mac() {
   # Install PHP
   curl -s https://php-osx.liip.ch/install.sh | bash -s 5.6
-  export PATH="/usr/local/php5-5.6.25-20160831-101628/bin:$PATH"
+  PHP_FOLDER=`find /usr/local -type d -name "php5-5.6*"`  # The folder name may change upon time
```

Comment:
Why not just use "/usr/local/php5"? https://php-osx.liip.ch/ clearly says php will be installed in /usr/local/php5.